app/views/jobs/getjob.html.erb

```diff
@@ -31,10 +31,10 @@
 </tr></table>
 
 <% if @cud.user.administrator?  then %>
-<li><kbd>notifyURL</kbd>: <kbd><%= @job[:rjob]["notifyURL"] %> </kbd>
+<li><kbd>notifyURL</kbd>: <%= @job[:rjob]["notifyURL"] %>
```

Comment:
I think Notify URL would be fine, especially since it's just making the property look nice.  But you can just make a decision and merge when you're ready.